content/docs/guides/integrations/flagger.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Integrate Flagger with OSM"
+description: "A simple demo showing how OSM integrates with Prometheus for metrics"
+aliases: "/docs/guides/integrations/demo_flagger"
+type: docs
+weight: 4
+---
+
+# Integrate Flagger with OSM
+
+To add additional automation around OSM's use of the [Service Mesh Interface](smi-spec.io) [Traffic Split](https://github.com/servicemeshinterface/smi-spec/blob/v0.6.0/apis/traffic-split/v1alpha4/traffic-split.md) functionality, OSM has provided integration with the [Flagger](https://www.weave.works/oss/flagger/) project developed by [WeaveWorks](https://www.weave.works/).
```

Comment:
(smi-spec.io) link doesn't work please fix it

Reply:
(https://github.com/servicemeshinterface/smi-spec/blob/v0.6.0/apis/traffic-split/v1alpha4/traffic-split.md)

OSM supports only v1alpha2 of traffic split maybe update this link to point to that ?